socrates/lib/registration/views/managementTables.pug

```diff
@@ -285,7 +285,7 @@ block content
                       td #{line.subscriber.country()}
                       td #{line.addon.pronoun()}
                       td #{formatList(registrationReadModel.roomTypesOf(line.member.id()))}
-                      td #{line.subscriber.isDiversity() ? 'yes' : ''}
+                      td #{line.subscriber.isDiversity() ? 'yes' : line.subscriber.addon().hasNonMalePronoun() ? 'check' : ''}
```

Comment:
Now this is the critical part of the functionality, and it had some issues already. If I were you, I would not hide this in a rather untestable place but instead put it in a function and test it with everything I have...